### PR TITLE
Moving fix for https://github.com/chef/chef-dk/issues/420 to master

### PR DESCRIPTION
**Pull request by tyler-ball**
\cc @danielsdeleo @jdmundrawala @fnichol

### CODE REVIEW

File: config/software/openssl-customization.rb [hunk 1]
```diff
@@ -60,8 +60,11 @@ def get_sanitized_rbconfig(config)
       # to pick up our script which find the CA bundle in omnibus installations and points SSL_CERT_FILE to it
       # if it's not already set
       source_openssl_rb = File.join(embedded_ruby_lib_dir, "openssl.rb")
-      File.open(source_openssl_rb, "a") do |f|
+      File.open(source_openssl_rb, "r+") do |f|
+        unpatched_openssl_rm = f.read
+        f.rewind
         f.write("\nrequire 'ssl_env_hack'\n")
+        f.write(unpatched_openssl_rm)
```
Comment (danielsdeleo) [inline config/software/openssl-customization.rb]: doesn't matter since the above line matches, but I think you want `unpatched_openssl_rb`

**Comment by danielsdeleo** [comment]
:+1:

File: config/software/openssl-customization.rb (hunk 1 above)
Comment (tyler-ball) [inline config/software/openssl-customization.rb]: too true

**Comment by jdmundrawala** [comment]
:shipit:

**Pull request merged**